include/aws/common/byte_buf.h

```diff
@@ -432,7 +432,7 @@ bool aws_byte_cursor_eq_c_str_ignore_case(const struct aws_byte_cursor *const cu
  * Case-insensitive hash function for array containing ASCII or UTF-8 text.
  */
 AWS_COMMON_API
-uint64_t aws_hash_array_ignore_case(const void *array, size_t len);
+uint64_t aws_hash_array_ignore_case(const void *array, const size_t len);
```

Comment:
Same deal on the `const size_t` being not useful, but don't worry about it for now, I'll go through and use clang-tidy to remove ALL of these at some point.